fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1554,120 +1554,121 @@ void schemaCoverage( std::string const& spath, bool covered ) {
 	}
 }
 
-bool schemaMatch( StatusObject const schema, StatusObject const result, std::string& errorStr, Severity sev, bool checkCoverage, std::string path, std::string schema_path ) {
+bool schemaMatch( json_spirit::mValue const& schemaValue, json_spirit::mValue const& resultValue, std::string& errorStr, Severity sev, bool checkCoverage, std::string path, std::string schemaPath ) {
```

Comment:
FYI: this change does what a previous comment suggested and recurses on values rather than objects. My memory is a little hazy now, but I think there was some issue with the schema I added with the previous version of this code.